pyDeltaRCM/init_tools.py

```diff
@@ -259,8 +259,10 @@ def create_domain(self):
         self.logger.info('Creating model domain')
 
         # ---- empty arrays ----
-        self.x, self.y = np.meshgrid(
-            np.arange(0, self.W), np.arange(0, self.L))
+        self.x, self.y = np.meshgrid(np.arange(0, self.W),
+                                     np.arange(0, self.L))
+        self.X, self.Y = np.meshgrid(np.arange(0, self.W+1)*self.dx,
+                                     np.arange(0, self.L+1)*self.dx)
```

Comment:
I know that this isn't the point of the PR, but does it make sense to have a scaled version of these things? If we are going to be shifting to using `xarray`, I could imagine this getting confusing.